src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -43,26 +44,31 @@ const SearchResults = (props) => {
         course: 4,
     });
     const [currentPage, setCurrentPage] = useState({
-        account: 1,
-        course: 1,
+        account: page,
+        course: page,
     });
+
+    const [moreResults, setMoreResults] = useState("");
+    const [lessResults, setLessResults] = useState("");
+
     const params = useParams();
 
-    const { accounts, courses, SearchQuery } = searchState;
 
-    useEffect(()=>{
-        return ()=>{
+
+    useEffect(() => {
+        return () => {
             api.resetSearchParams();
         }
-    },[]);
+    }, []);
 
     const numberOfResults = () => {
         switch (searchState.primaryFilter) {
             case SEARCH_ALL: {
-                return accounts.length + courses.length;
+                return count;
```

Comment:
Keep in mind. this is only the account count - you'll also need to add the course count when that's done developing. So be sure to add a TODO comment so you don't forget